packages/@lwc/engine-core/src/framework/__tests__/Shadow-getter.spec.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { LightningElement } from '../../../src';
+
+Object.freeze(LightningElement); // because we freeze before exporting for users as well
+
+describe('Shadow', () => {
+    it('should be true for LightningElement', () => {
+        expect(LightningElement.shadow).toBeTruthy();
```

Comment:
The 2 tests should explicitly check if the `shadow` value is `true` or `false`. The test should fail if the returned value is not a boolean.

Reply:
Done